oncopipe/oncopipe/__init__.py

```diff
@@ -207,23 +192,79 @@ def coerce_namedlist_to_string(obj):
     else:
         dest_dir, dest_file = os.path.split(dest)
     os.makedirs(dest_dir, exist_ok=True)
-
     dest = os.path.join(dest_dir, dest_file)
+    
+    return [src, dest]
```

Comment:
For future reference, you don't need parentheses or brackets when returning multiple values in Python. By default, a tuple (_i.e._ an immutable list) will be returned. You don't need to change this here though because this works fine. 

I've included an example below that demonstrates how this works and how you can use unpacking in Python to assign both values to separate variables in one go. 

```
>>> def foo():
...     return 1, 2
...
>>> foo()
(1, 2)

>>> a, b = foo()
>>> a
1
>>> b
2
```

Reply:
Thanks!